docs/en/table_design/table_types/primary_key_table.md

```diff
@@ -1,159 +1,212 @@
+:::
 
-  - The primary key must be created on columns on which unique constraints are enforced, and the names of the primary key columns cannot be changed.
+</TabItem>
 
-  - The primary key columns can be any of the following data types: BOOLEAN, TINYINT, SMALLINT, INT, BIGINT, LARGEINT, STRING, VARCHAR, DATE, and DATETIME. However, the primary key columns cannot be defined as `NULL`.
+<TabItem value="example2" label="fully in-memory primary key index">
 
-  - The partition column and the bucket column must participate in the primary key.
+When `enable_persistent_index` is set to `false`, the primary key indexes are not persisted to the disk,  that is, the primary key indexes are fully stored in memory. During loading, the primary key indexes of tablets related to the data loaded will be loaded into memory, which may result in higher memory consumption. (If a tablet has not had data loaded for a long time, its primary key index will be released from memory).
 
-  - The number and total length of primary key columns must be properly designed to save memory. We recommend that you identify columns whose data types occupy less memory and define those columns as the primary key. Such data types include INT and BIGINT. We recommend that you do not let a column of the VARCHAR data type to participate in the primary key.
+When using the fully in-memory primary key indexes, it is recommended to follow the following guidelines when designing the Primary Key to control memory usage of the primary key indexes:
 
-  - Before you create the table, we recommend that you estimate the memory occupied by the primary key index based on the data types of the primary key columns and the number of rows in the table. This way, you can prevent the table from running out of memory. The following example explains how to calculate the memory occupied by the primary key index:
-    - Suppose that the `dt` column, which is of the DATE data type that occupies 4 bytes, and the `id` column, which is of the BIGINT data type that occupies 8 bytes, are defined as the primary key. In this case, the primary key is 12 bytes in length.
+- The number and total length of Primary Key columns must be properly designed. We recommend that you identify columns whose data types occupy less memory and define those columns as the Primary Key, such as INT and BIGINT, not VARCHAR.
+- Before you create the table, we recommend that you estimate the memory occupied by the primary key indexes based on the data types of the Primary Key columns and the number of rows in the table. This way, you can prevent running out of memory. The following example explains how to calculate the memory occupied by the primary key indexes:
+  - Suppose that the `dt` column, which is of the DATE data type that occupies 4 bytes, and the `id` column, which is of the BIGINT data type that occupies 8 bytes, are defined as the Primary Key. In this case, the Primary Key is 12 bytes in length.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/19de5c2a-4fa6-4d1a-8112-a98d9523216f)
-> and